pages/en/lb4/Server.md

```diff
@@ -0,0 +1,40 @@
+---
+lang: en
+title: 'Server'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Server.html
+summary:
+---
+
+## Overview
+
+The [Server](https://apidocs.strongloop.com/@loopback%2fcore/#Server) interface defines the minimal required functions (start and stop) to implement for a LoopBack application. Servers in LoopBack 4 are used to represent implementations for inbound transports and/or protocols such as REST over http, gRPC over http2, graphQL over https, etc. They typically listen for requests on a specific port, handle them, and return appropriate responses. A single application can have multiple server instances listening on different ports and working with different protocols.
```

Comment:
Based on all discussions I have participated, I think

> Servers in LoopBack 4 are used to represent implementations for inbound transports and/or protocols such as REST over http, gRPC over http2, graphQL over https, etc. They typically listen for requests on a specific port, handle them, and return appropriate responses.

is a proper definition with no contradiction. 

> A single application can have multiple server instances listening on different ports and working with different protocols.

IIRC in the current release we decide to avoid dealing with multiple servers bound to one app, focus on the use case "one app only has one server". Better confirm with @kjdelisle about it....

Reply:
The current implementation allows multiple "servers"; it's the RestServer's implementation of things like `@api` and other tags that don't quite fit with the current implementation (storing their values in the top-level metadata container is why this is an issue)

Reply:
Thanks @jannyHou @kjdelisle. I'll keep it as is, but if you feel strongly about removing it since we don't have concrete examples for it, I don't mind doing so.